src/datasources/transaction-api/transaction-api.service.ts

```diff
@@ -284,4 +285,33 @@ export class TransactionApi implements ITransactionApi {
       throw this.httpErrorFactory.from(error);
     }
   }
+
+  async getTokenInfo(address: string): Promise<Page<TokenInfo>> {
```

Comment:
nitpick: `getToken` seems to fit better. Any reason for having `Info` there?

(the same applies to the entity and `getTokensInfo`)

Reply:
I changed it in [16eda59](https://github.com/5afe/safe-client-gateway-nest/pull/178/commits/16eda598ca12e2c19b144794bd4a8812dae7d5db)

I also changed `TokenInfo` to `Token` as we discussed.